main.go

```diff
@@ -155,7 +155,12 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "NestedCluster")
 		os.Exit(1)
 	}
+
 	// +kubebuilder:scaffold:builder
+	if err := (&infrastructurev1.NestedCluster{}).SetupWebhookWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create webhook", "webhook", "NestedCluster")
+		os.Exit(1)
+	}
```

Comment:
Sorry to be pedantic about this but can you move it above `// +kubebuilder:scaffold:builder`

Reply:
Other than this, this PR looks and works great. Once this is done we can lgtm and approve it.

Reply:
thanks for comment, just updated , appreciate the review~